.golangci.yml

```diff
@@ -8,7 +8,7 @@ linters-settings:
     default-signifies-exhaustive: true
   forbidigo:
     forbid:
-      - 'fmt\.Print.*'            # Prevent debug logging
+      - 'fmt\.Print.*(# Avoid debug logging)?'
```

Comment:
forbidigo supports UX messages inside regexp

Reply:
forbidigo only supports checking single identifiers. It cannot check function calls with arguments in a single regexp.